argon2/src/block.rs

```diff
@@ -44,8 +44,12 @@ macro_rules! permute {
     };
 }
 
+#[cfg(any(target_arch = "x86_64"))]
+cpufeatures::new!(avx2_cpuid, "avx2");
+
 /// Structure for the (1 KiB) memory block implemented as 128 64-bit words.
 #[derive(Copy, Clone, Debug)]
+#[cfg_attr(test, derive(PartialEq))]
```

Comment:
I think it will be better to write `assert_eq!(result.0, result_av2.0);` than use this `cfg`.